documentation/onboard-dpg-in-sdkautomation/java/README.md

```diff
@@ -0,0 +1,38 @@
+# Add Autorest Configuration of Java SDK 
+
+## Parameter Description
+
+- `<ServiceName>`: The RP name, which is usually same as rp name in swagger.
+- `<PackageName>`: Java package name.
+
+
+## Single Client
+If you want to generate sdk with single client, please copy the following configuration into spec PR comment.
+~~~
+# azure-sdk-for-java
+``` yaml
+output-folder: sdk/<ServiveName>/<PackageName>
+require:
+ - specification/<RPName>/dataplane/readme.md
+```
+~~~
+- `output-folder`: The relative path of destination to generate SDK.
+- `require`: The item of the value is the relative path of spec readme.md file.
+
+## Multi Client
+If you want to generate sdk with multi client, please copy the following configuration into spec PR comment.
+~~~
+# azure-sdk-for-java
+``` yaml
+tag: false
+output-folder: sdk/<ServiveName>/<PackageName>
+require:
+ - specification/<ServiveName>/dataplane/readme.md
```

Comment:
@dw511214992 
Do you require script to make URL like https://raw.githubusercontent.com/openapi-env-test/azure-rest-api-specs/6b7d7b3cc765f3fe1dc22e802a8e30c444ce3403/specification/deviceupdate/data-plane/readme.md from this `specification/<ServiveName>/dataplane/readme.md` in comment?

If so, can you make it clear in the doc the composition of the URL from components in `input.json`.

Reply:
I misunderstood, local path to the readme.md will do. That particular autorest.md is only to be used in CI, not anywhere else.

Reply:
In generating autorest.md, please use local path in for the required readme.md because sometimes we cannot get the http url in the procedure of swagger PR, such as the readme.md is still in customer's private repo.

Reply:
As I mentioned in another comment I do think this is going to become an issue for us once we start to extend this to creating PRs in the language repos which will need to be full url's. So lets think about this a little more to see how we can handle that.

Reply:
Yes, but I think in swagger PR CI, it's fine to use local path. For now, I think the only way is asking service team to change it manually when they use the generated code as a base branch. In the release pipeline of powerapp workflow, we can get a valid http url to readme.md because the swagger PR should be merged to public spec repo.